components/Avatar/Avatar.scss

```diff
@@ -0,0 +1,73 @@
+@import '../util/include-responsive.scss';
+
+.defaultAvatarXsmall,
```

Comment:
I guess I have new styles coming in here too?

Reply:
Good to note.  Can you rebase after merging on this one too?

Reply:
Yup